doc/z-chap03.xml

```diff
@@ -121,17 +160,8 @@
 
   A bipartition &x; of degree &n; is called <E>planar</E> if there does not exist
```

Comment:
if there **do** not exist

(sorry, I know you haven't actually modified this bit)